core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -115,10 +115,20 @@ export function Container({ containerRef }: ContainerProps) {
                 title: contentlet.widgetTitle || contentlet.title,
                 inode: contentlet.inode
             },
+            treeNode: {
+                containerId: container.identifier,
+                contentId: contentlet.identifier,
+                relationType: container.uuid,
+                variantId: container.variantId,
+                personalization: getPersonalization(viewAs.persona),
+                treeOrder,
+                pageId
+            },
```

Comment:
we already has this information in the container in LN122, check if we can make use of that.

Reply:
Done!